doc/content/source/materials/ConcreteThermalMoisture.md

```diff
@@ -360,14 +381,13 @@ $P_{vs}$ = saturate vapor pressure $= P_{atm}\exp\left(4871.3\frac{T-100}{373.15
 $P_{atm}$ = standard atmospheric pressure $= 101.325 Pa$ \\
 $D_h$ = moisture diffusivity (also referred as humidity diffusivity), $kg/(m^2\cdot s)$\\
 $D_{ht}$= coupled moisture diffusivity under the influence of a temperature gradient, $kg/(m^2\cdot s\cdot K)$\\
-$W_d$= total mass of free evaporable water released into the pores by dehydration of the cement paste\\
 $t$  = time, $s$
 
 The term on the left side of [!eqref](moisture_governing) represents time-dependent effects, and is provided by [ConcreteMoistureTimeIntegration](ConcreteMoistureTimeIntegration.md). The first term on the right side of [!eqref](moisture_governing) represents Fickian diffusion, and the second term represents Soret diffusion. These are both provided by [ConcreteMoistureDiffusion](ConcreteMoistureDiffusion.md). The third term on the right hand side of this equation represents a source due to dehydrated water, and is provided by [ConcreteMoistureDehydration](ConcreteMoistureDehydration.md).
 
 Moisture diffusivity $D_h$ depends on the relative humidity, $H$. Thus the moisture diffusion governing equation is highly nonlinear. The following sections describes in detail the constitutive models for moisture diffusivity.
 
-### Moisture diffusivity
+### Moisture diffusivity and coupling parameters with heat diffusion
```

Comment:
Suggested more clear wording:

Coefficients for moisture diffusivity and coupling with heat diffusion